src/boot/efi/util.c

```diff
@@ -769,6 +770,108 @@ EFI_STATUS make_file_device_path(EFI_HANDLE device, const char16_t *file, EFI_DE
         return EFI_SUCCESS;
 }
 
+#define QEMU_KERNEL_LOADER_FS_MEDIA_GUID \
+        { 0x1428f772, 0xb64a, 0x441e, {0xb8, 0xc3, 0x9e, 0xbd, 0xd7, 0xf8, 0x93, 0xc7 }}
+
+#define QEMU_BOOT_ORDER_GUID \
+        { 0x668f4529, 0x63d0, 0x4bb5, {0xb6, 0x5d, 0x6f, 0xbb, 0x9d, 0x36, 0xa4, 0x4a }}
```

Comment:
Please make this a pointer of `EFI_GUID`, like the other XXX_GUID definitions:
```
# define QEMU_KERNEL_LOADER_FS_MEDIA_GUID \
         &(const EFI_GUID) { ... }
```

Reply:
Pleas no. This was a terrible concept when it was introduced and prevents usage of these in other contexts. For example, when trying to assemble struct: https://github.com/systemd/systemd/blob/957dfcc96d5f7febc58c3d1d6f969bda854fb98b/src/boot/efi/secure-boot.c#L89
EFI_GUID is just 128bits, that's cheap to assemble when needed.

Reply:
So, can you please agree how you want it?  I don't care much, but current code doesn't embed them in other structs so it doesn't make much of a difference either way.

Reply:
Please follow @medhefgo's suggestion, and ignore my comment then. Sorry for the noise.